### PR TITLE
DESENG-148 Fix chown errors

### PR DESCRIPTION
**Pull request by Baelx**
Note that docker-entrypoint.sh and Dockerfile have been mostly taken from the forked repo mentioned in README.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,65 @@
+FROM php:8.0-apache
```
Comment (ShawnTurple) [inline Dockerfile:1]: i would consider using a more specific version, https://hub.docker.com/layers/php/library/php/8.0.20-apache/images/sha256-9394291bd7b1bf1c03ee43e613107fe1563582ac853f441ce8a7abf2a4128502?context=explore also look into github actions to determine when this changes.   
Considerations for future, but not required now.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+ENV DOWNLOAD_URL https://download.limesurvey.org/latest-stable-release/limesurvey5.3.19+220607.zip
```
Comment (ShawnTurple) [inline Dockerfile:3]: fyi 5.3.20 is the latest.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,18 @@
-LimeSurvey Docker Compose
+# LimeSurvey Docker Compose
```
Comment (ShawnTurple) [inline README.md]: Might want to consider adding versions in readme as a quick glance, future considerations.

File: docker-compose.yml [hunk 4]
```diff
@@ -5,12 +5,12 @@ networks:
   lime_deploy:
 volumes:
   database:
-  plugins:
   upload:
+  plugins:
   config:
+  themes:
```
Comment (ShawnTurple) [inline docker-compose.yml:11]: what are these docker volumes for??

File: docker-entrypoint.sh [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+    if ! [ -e application/config/config.php ]; then
+        echo >&2 "No config file in $(pwd) Copying default config file..."
+		#Copy default config file but also allow for the addition of attributes
+awk '/lime_/ && c == 0 { c = 1; system("cat") } { print }' application/config/config-sample-mysql.php > application/config/config.php <<'EOPHP'
+'attributes' => array(),
+EOPHP
+    fi
+
+	# Install BaltimoreCyberTrustRoot.crt.pem
+	if ! [ -e BaltimoreCyberTrustRoot.crt.pem ]; then
```
Comment (ShawnTurple) [inline docker-entrypoint.sh:84]: this is a bit interesting.

**Review by ShawnTurple — APPROVED** [review]

**Comment by Baelx** [comment]
@ShawnTurple Thanks for the comments. I've added the versions into the README. The original ticket had no versioning requirements other than a working Limesurvey 5.x.x. The volumes(except for the mounted theme directory) were all in the original docker compose file and I assume are necessary for persisting changes to those app directories.

**Pull request merged**